packages/aws-cdk-lib/aws-globalaccelerator/lib/accelerator.ts

```diff
@@ -82,6 +142,18 @@ export class Accelerator extends cdk.Resource implements IAccelerator {
    */
   public readonly dnsName: string;
 
+  /**
+   * IP addressess associated with the accelerator.
+   */
+  public readonly ipAddresses?: string[];
+
+  /**
+   * The IP address type that an accelerator supports.
+   *
+   * For a standard accelerator, the value can be IPV4 or DUAL_STACK.
+  */
+  public readonly ipAddressType?: IpAddressType;
```

Comment:
```suggestion
  /**
   * The IP address type that an accelerator supports.
   *
   * For a standard accelerator, the value can be IPV4 or DUAL_STACK.
   */
  public readonly ipAddressType?: IpAddressType;
```

But, Is it necessary to have even the `ipAddressType` as a property?